content/docs/command-reference/repro.md

```diff
@@ -36,9 +39,9 @@ and caches relevant <abbr>data artifacts</abbr> along the way.
 💡 For convenience, a Git hook is available to remind you to `dvc repro` when
 needed after a `git commit`. See `dvc install` for more details.
 
-There's a few ways to restrict the stages that will be regenerated by this
-command: by specifying stage file `targets`, or by using the `--single-item`,
-`--cwd`, or other options.
+There are a few ways to restrict the stages that will be regenerated by this
+command: by specifying stages or `.dvc` files to reproduce, or by using the
```

Comment:
can it accept an output? 

stages -> stage names

.dvc - probably it's better not to metion this? what is the use case for this? can it be done w/o repro?

Reply:
> can it be done w/o repro?

Yes, `.dvc` files can easily be updated via `dvc add`.

Reply:
> can it accept an output?

No, repro/run don't support this nor granularity (files inside tracked dirs)

> .dvc - what is the use case for this?

Same as using `dvc add` again on a changed data file.

Keeping it in this paragraph because it's not an edge case to have the raw data as .dvc files at the beginning of a pipeline and `dvc repro` will by default update these if the data changed.

Updated in #1615.

Reply:
Even if it the case (repro updates .dvc) - it's a side effect, not a main goal of the `dvc repro` ... `dvc repro something.dvc` for `dvc add`-ed files is confusing and not worth complicating docs for I think.

Reply:
> dvc repro something.dvc for dvc add-ed files is confusing

Good point. Generalized in ebc15605.